include/controller.h

```diff
@@ -89,9 +89,10 @@ class Controller
     float tau_;
   };
 
-  vector_t run_pid_loops(float dt, const Estimator::State& state, const control_t& command, bool update_integrators);
-
   ROSflight& RF_;
+
+  vector_t run_pid_loops(int32_t dt, const Estimator::State& state, const control_t& command, bool update_integrators);
```

Comment:
Are we going to allow negative dts? If not maybe we should change dt to a uint32_t to enforce that